Formula/kumactl.rb

```diff
@@ -0,0 +1,30 @@
+class Kumactl < Formula
+  desc "Kuma control plane command-line utility"
+  homepage "https://kuma.io/"
+  url "https://github.com/Kong/kuma.git",
+      :tag      => "0.4.0",
+      :revision => "a37aaadf84e97fa9e5135c31353b51ba6a2245f3"
```

Comment:
let's use tarball if possible
```suggestion
  url "https://github.com/Kong/kuma/archive/0.4.0.tar.gz"
  sha256 "a245aa3d5afe76a6254cc44a6dd8f35d17d0ebd0f15db25e2345062f93cd036c"
```